HoneyBadgers.Logic/Menu.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using static System.Console;
+
+namespace HoneyBadgers.Logic
+{
+    class Menu
+    {
+        private int SelectedIndex { get; set; }
+        private string[] Options { get; }
```

Comment:
czemu Options i Prompt są bez setter'ów?

Reply:
jak próbuje w tych dwóch dać set to pokazuje mi taki komunikat :"Auto-property can be made get-only"

Reply:
zdefiniowałeś to jakoś, może przez przypadek? [https://www.jetbrains.com/help/resharper/Making_AutoProperties_GetOnly.html](https://www.jetbrains.com/help/resharper/Making_AutoProperties_GetOnly.html)

Reply:
probówałem dodać to od nowa ale jest ten sam efekt, a wpisywałem ręcznie specjalnie ;/